app/src/main/java/org/simple/clinic/summary/PatientSummaryEvent.kt

```diff
@@ -9,7 +10,12 @@ import java.util.UUID
 
 sealed class PatientSummaryEvent : UiEvent
 
-data class PatientSummaryProfileLoaded(val patientSummaryProfile: PatientSummaryProfile) : PatientSummaryEvent()
+data class PatientSummaryProfileLoaded(val patientSummaryProfile: PatientSummaryProfile) : PatientSummaryEvent() {
+
+  override fun toString(): String {
+    return "PatientSummaryProfileLoaded(patientSummaryProfile=$REDACTED_STRING)"
+  }
+}
```

Comment:
Why don't we redact the `toString()` of `PatientSummaryProfile` and `MedicalOfficer` instead? That way, we don't need to do this at every usage site.

Reply:
I have no idea why I didn't just do that, that makes a lot of sense 😅